sidebars.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
           "get-started/installing-cardano-wallet",
         ],
       },
+      "get-started/air-gap",
+      "get-started/secure-workflow",
```

Comment:
IMO - the secure-workflow/air-gap might be better suited under best-practices (there was a placeholder created in #658 in [sidebar.js](https://github.com/sanskys/developer-portal/blob/9363bff6ea66f2b76cb1ded261fccc6f8ab92286/sidebars.js#L159) ), but may be if there are multiple pages - can be top-level folder called `Security Best Practices` with pages underneath?

Reply:
`stake-pool-guide/best-practices` in that pending PR is under the Stake Pool Guide hierarchy, but both of these pages are applicable to anyone (not just SPOs) who creates transactions manually: developers issuing minting transactions, etc.  To put these pages in any Stake Pool hierarchy would cause developers in general and token minters to overlook it or not use it.

I'm OK with putting these pages into a "Security Best Practices" section outside the body of SPO material.  Maybe keep the pages where they are now in linear order, and just put them under a `Security Best Practices` heading?

Reply:
p.s. OK I can see that is exactly what you are suggesting 🤪 ... will push it out that way & see how it works 😎

Reply:
This is done & I think it looks/works better this way.